clients/src/main/java/org/apache/kafka/clients/consumer/internals/DefaultBackgroundThread.java

```diff
@@ -168,28 +171,41 @@ public DefaultBackgroundThread(final Time time,
                             logContext);
             CoordinatorRequestManager coordinatorRequestManager = null;
             CommitRequestManager commitRequestManager = null;
+            HeartbeatRequestManager heartbeatRequestManager = null;
+            MembershipManager membershipManaber = null;
 
+            // TODO: consolidate groupState and memberState
             if (groupState.groupId != null) {
                 coordinatorRequestManager = new CoordinatorRequestManager(
-                        this.time,
-                        logContext,
-                        retryBackoffMs,
-                        retryBackoffMaxMs,
-                        this.errorEventHandler,
-                        groupState.groupId);
+                    this.time,
+                    logContext,
+                    retryBackoffMs,
+                    retryBackoffMaxMs,
+                    this.errorEventHandler,
+                    groupState.groupId);
                 commitRequestManager = new CommitRequestManager(
-                        this.time,
-                        logContext,
-                        subscriptionState,
-                        config,
-                        coordinatorRequestManager,
-                        groupState);
+                    this.time,
```

Comment:
Indentation? (I guess it shouldn't be the same as in the requestManager down below)

Reply:
🤦